k8s/aat/common/family-public-law/fpl-case-service.yaml

```diff
@@ -27,52 +27,11 @@ spec:
       image: hmctspublic.azurecr.io/fpl/case-service:prod-f4f18a39
       ingressHost: fpl-case-service-aat.service.core-compute-aat.internal
       environment:
-        IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
-        IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-        CORE_CASE_DATA_API_URL: http://ccd-data-store-api-aat.service.core-compute-aat.internal
-        DOCUMENT_MANAGEMENT_URL: http://dm-store-aat.service.core-compute-aat.internal
-        CCD_UI_BASE_URL: https://www-ccd.aat.platform.hmcts.net
-        DOCMOSIS_TORNADO_URL: https://docmosis-development.platform.hmcts.net
-        RD_PROFESSIONAL_API_URL: http://rd-professional-api-aat.service.core-compute-aat.internal
-        SEND_LETTER_URL: http://rpe-send-letter-service-aat.service.core-compute-aat.internal
-        FEES_REGISTER_API_URL: http://fees-register-api-aat.service.core-compute-aat.internal
-        PAYMENT_API_URL: http://payment-api-aat.service.core-compute-aat.internal
-        IDAM_CLIENT_ID: fpl_case_service
-        IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-*.service.core-compute-aat.internal/oauth2/callback
```

Comment:
in chart global values it is different:
https://fpl-case-service-aat.service.core-compute-aat.internal/oauth2/callback
it is override then in 
values.aat.template.yaml
with correct one, but i think 
values.aat.template.yaml
is not taken into account when released version is deployed by flux, this can be potential problem

Reply:
Do we need `-*` on AAT ? It is needed on preview because each PR will have unique URL and aat staging slot , we will have  `-staging` , but on running AAT instance , it should always be -aat anyway (which is covered by values.yaml). Please correct me if i am wrong.

Reply:
Do we need -* on AAT ? It is needed on preview because each PR will have unique URL and aat staging slot , we will have -staging , but on running AAT instance , it should always be -aat anyway (which is covered by values.yaml). Please correct me if i am wrong.

Reply:
preview use idam from aat but each PR should self register its own redirect url, so hopefully it should work